### ISSUE TITLE
Send email summary report to user

### ISSUE THREAD

**Issue by melikecode** [charlottebrf/DoctorsWhoCode#23]
- I want to be emailed a report at a pre-determined time each day, showing me time spent on activities in text form (plain or HTML)
	• Does the email include a reminder of targets that were set, or any suggestions based on that?
	• Where is the user asked for their email address (e.g. on a "reporting" page)?
	• Can the user determine what time the report is sent (e.g. on a "reporting" page)?
	• If email address and reporting time are captured, this should be stored with user's details
	• Could this be done as a weekly summary instead/also?
	• Does the email tell me if I spent more/less on activities where I specified a time target

**Comment by charlottebrf** [comment]
@ajh146 So does this require mailgun?

**Comment by melikecode** [comment]
Yep and or pony (  mentioned in one of the course notes for Ruby)
